app/services/iteration_service.rb

```diff
@@ -0,0 +1,59 @@
+class IterationService
+  DAYS_IN_WEEK = (1.week / 1.day)
+  FIELDS = [:id, :estimate, :accepted_at, :story_type]
+
+  attr_reader :project, :stories
+
+  delegate :start_date, :start_date=,
+    :iteration_length, :iteration_length=,
+    :iteration_start_day, :iteration_start_day=,
```

Comment:
I'll keep these exposed for the time being. We should be able to simulate how a project goes by changing the iteration length or start day. And actually changing those won't commit anything to the database so it's no big deal.